  INFO root [2018-07-09 08:19:42,964] - 
  INFO root [2018-07-09 08:19:42,964] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-09 08:19:42,964] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2018-07-09 08:19:42,964] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [2018-07-09 08:19:42,964] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2018-07-09 08:19:42,964] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.3.16.255 ( 10.3.16.255:30099 )
  INFO root [2018-07-09 08:19:42,980] - NetworkInterface::initialize(INTERNAL): address 10.3.16.255:30099, SOMAXCONN=5.
  INFO root [2018-07-09 08:19:42,980] - ---- interfaces Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: A3D4913C83C4082A29723B2DA267A5B0. Config: Debug X86. Built: 16:26:48 Apr 30 2018. AppID: 3000. UID: 10103. PID: 10968 ----
  INFO root [2018-07-09 08:19:42,980] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [2018-07-09 08:19:42,980] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-09 08:19:42,980] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-09 08:19:43,174] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-09 08:19:43,174] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-09 08:19:43,174] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-09 08:19:43,189] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-09 08:19:43,189] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-09 08:19:43,189] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-09 08:19:43,189] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-09 08:19:44,330] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-09 08:19:46,142] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:26:41) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-07-09 08:19:46,157] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-09 08:19:46,754] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-09 08:19:47,019] - TelnetServer server is running on port 33000
  WARN root [2018-07-09 08:19:47,019] - interfaces.cpp:146: Profile scriptCall took 0.26 seconds
  INFO root [2018-07-09 08:19:47,519] - ---- interfaces is running ----
 DEBUG root [2018-07-09 08:19:47,519] - Components::process(): Request for the process of identity...
  INFO root [2018-07-09 08:19:55,080] - Components::process(): Found all the components!
S_INFO root [2018-07-09 08:22:53,267] - onRequestAccountLogin: registerName=cxx
S_INFO root [2018-07-09 08:23:42,467] - onRequestCreateAccount: registerName=cxx2017
S_INFO root [2018-07-09 08:23:44,667] - onRequestAccountLogin: registerName=cxx2017
  WARN root [2018-07-09 08:25:25,913] - TCPPacketReceiver::processPendingEvents(127.0.0.1:12888): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
